cluster/images/etcd/migrate/versions.go

```diff
@@ -124,15 +124,7 @@ func ParseEtcdVersionPair(s string) (*EtcdVersionPair, error) {
 	return &EtcdVersionPair{version, storageVersion}, nil
 }
 
-// MustParseEtcdVersionPair parses a "<version>/<storage-version>" string to an EtcdVersionPair
-// or panics if the parse fails.
-func MustParseEtcdVersionPair(s string) *EtcdVersionPair {
```

Comment:
Same, it's being used https://cs.k8s.io/?q=MustParseEtcdVersionPair&i=nope&files=&excludeFiles=&repos=kubernetes/kubernetes

Reply:
please see https://golangci-lint.run/usage/false-positives/ on how to add `//nolint` directives to tell linter to ignore this